main.go

```diff
@@ -8,11 +8,10 @@ import (
 	"github.com/bndw/pick/config"
 )
 
-// Injected in build
-var version string
+const Version = "v0.5.0"
```

Comment:
Well — it was exported before commit 219084040cc7d2c956559794f954f47dfcacb645 which I reverted here. It's unexported now.
The version now is in its own file and doesn't include the 'v' prefix.